### PR TITLE
Quiz play MVP

### PR DESCRIPTION
**Pull request by risakawagoe**
Implemented Deplyable MVP for quiz play. Quiz play screens include:

- Quiz Configuration
- Quiz Play (goes through each question) *only displays the question for MVP
- Quiz Result (TODO: implement score calculation

### CODE REVIEW

File: frontend/src/components/quizzerhub_components/quiz_card.tsx [hunk 1]
```diff
@@ -7,6 +7,20 @@ interface Props {
     quiz: Quiz
     selectQuiz: (id: string) => void
 }
+export const getColorByAvgScore = (score: number) => {
```
Comment (jerrydngzh) [inline frontend/src/components/quizzerhub_components/quiz_card.tsx:10]: picky: a switch statement could be used here

**Review by jerrydngzh — APPROVED** [review]
lgtm,

some things im just being picky about in terms of syntax is for react components its more idiomatic and visually contrasting to have functional components 
i.e.
```jsx
export function ComponentName(props){
 return (...)
}
```
over arrow functions as components
```jsx
const component = (props) => {
 return (...)
}
```

It gets a bit hard to read large lines of code when now the main component starts look like arrow functions, where arrow functions idiomatically lends themselves as an operation to do something vs something that is returning a "component"

otherwise everything is good

File: frontend/src/components/quizzerhub_components/quiz_card.tsx (hunk 1 above)
Comment (risakawagoe) [reply frontend/src/components/quizzerhub_components/quiz_card.tsx:10]: i wasn't sure how to do it but do you mean like this?
```
switch (true) {
    case score > 85:
        return 'green';
    case score > 70:
        return 'blue';
    case score > 50:
        return 'yellow';
    case score > 30:
        return 'orange';
    default:
        return 'red';
}
```

Comment (jerrydngzh) [reply frontend/src/components/quizzerhub_components/quiz_card.tsx:10]: yep

**Comment by risakawagoe** [comment]
Thanks for the comments, Re: component naming and usage of lambda exp., let me refactor that in later updates since this PR will basically be the baseline for deployment (at least for frontend) excluding changes for containerization and deployment config etc.
But will try to refactor as long as there's time remaining till submission.

File: frontend/src/components/quizzerhub_components/quiz_card.tsx (hunk 1 above)
Comment (jerrydngzh) [reply frontend/src/components/quizzerhub_components/quiz_card.tsx:10]: ah i see, actually i think you can ignore my comment on the switch case then, didnt properlly read what the conditionals were doing

**Comment by jerrydngzh** [comment]
oh yeah, alot of it is just syntactical refactoring. ive noticed arrow components as the main component being returned showing up in docs alot so was curious about your opinion on this as well

**Pull request merged**